apps/web/src/components/layout/components/SideNav.tsx

```diff
@@ -1,7 +1,13 @@
-import { useState } from 'react';
+import { useState, useContext } from 'react';
+import { useQuery, useQueryClient } from 'react-query';
 import { Navbar } from '@mantine/core';
+import { IEnvironment } from '@novu/shared';
+import { getMyEnvironments, getCurrentEnvironment } from '../../../api/environment';
+import { api } from '../../../api/api.client';
 import { NavMenu, SegmentedControl } from '../../../design-system';
 import { Activity, Bolt, Box, Settings, Team } from '../../../design-system/icons';
+import { AuthContext } from '../../../store/authContext';
+import { act } from 'react-dom/test-utils';
```

Comment:
Is this a mistake or why is this imported here?

Reply:
Idk.. did you add it? Lol

Reply:
haha now :P